### PR TITLE
Revert "Revert "Tool list package id feature doc update""

### PR DESCRIPTION
**Pull request by IEvangelist**
Reverts dotnet/docs#32002, coming full circle—getting back to the desired state of having this PR not yet merged, but approved.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/tools/dotnet-tool-list.md](https://github.com/dotnet/docs/blob/0dd63a3c3895b97548a37d4cfbda6ba4fbdd6696/docs/core/tools/dotnet-tool-list.md) | [dotnet tool list](https://review.learn.microsoft.com/en-us/dotnet/core/tools/dotnet-tool-list?branch=pr-en-us-32003) |
| [docs/core/tools/global-tools.md](https://github.com/dotnet/docs/blob/0dd63a3c3895b97548a37d4cfbda6ba4fbdd6696/docs/core/tools/global-tools.md) | [How to manage .NET tools](https://review.learn.microsoft.com/en-us/dotnet/core/tools/global-tools?branch=pr-en-us-32003) |

### CODE REVIEW

**Review by tdykstra — APPROVED** [review]

**Comment by JiaqiWang18** [comment]
FYI, the [feature pull request](https://github.com/dotnet/sdk/pull/28333) has been merged today.

**Comment by luisquintanilla** [comment]
@IEvangelist is this one ready for review?

File: docs/core/tools/global-tools.md [hunk 1]
```diff
@@ -189,6 +189,20 @@ dotnetsay       2.1.3        dotnetsay      /home/name/repository/.config/dotnet
 
 As shown in the preceding example, the list shows local tools. To see global tools, use the `--global` option. To see tool-path tools, use the `--tool-path` option.
 
+To list a specific tool, use the [dotnet tool list <PACKAGE_ID>](dotnet-tool-list.md) command:
+
+```dotnetcli
+dotnet tool list dotnetsay
+```
+
+The output will be only contain information of that tool if it is installed, similar to the following example:
```
Comment (tdykstra) [inline docs/core/tools/global-tools.md:198]:
```suggestion
The output will only list that tool if it's installed, similar to the following example:
```

**Review by tdykstra — APPROVED** [review]
One additional suggestion, then it's ready to go.

**Pull request merged**